flexmeasures/data/services/forecasting.py

```diff
@@ -108,6 +108,7 @@ def create_forecasting_jobs(
                 custom_model_params=custom_model_params,
             ),
             connection=current_app.queues["forecasting"].connection,
+            ttl=24 * 60 * 60,  # try up to one day
```

Comment:
Looks to me like this should be a config setting rather than be hard coded. I'd also prefer to set the config setting as a timedelta (like `FLEXMEASURES_PLANNING_TTL`) and calling `.total_seconds()` on it here.

Reply:
It looks that way to me too, but I'm afraid of having too many config settings.